.clang-format

```diff
@@ -88,7 +88,7 @@ PenaltyExcessCharacter: 4
 PenaltyReturnTypeOnItsOwnLine: 100
```

Comment:
Do you use format for single files? 
I want to reformat the whole repository but there are some issues with compilation so I'm confused how does it work on your side.

Reply:
Hm, I didn't receive a penalty there.
Also, it already was there and the only thing which I changed was the `Standard:` `value: `Cpp11` → `c++11` (previous value wasn't recognized)

Reply:
mhm. 
Please do not format whole codebase within this PR. 

I want to do it at some point, but it makes ~80k lines change :D